templates/csharp/xplat/AvaloniaTest.iOS/AvaloniaTest.iOS.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>FrameworkParameter-ios</TargetFramework>
+    <TargetFramework>FrameworkParameter-ios16.1</TargetFramework>
```

Comment:
It shouldn't be updated here.
Instead, I suppose, SDK in this repository should be upgraded (global.json, pipeline settings).

Reply:
With SDK 7.0.102 used in pipeline this error still occures:

> error NU1202: Package Avalonia.iOS 11.0.0-preview5 is not compatible with net7.0-ios16.0 (.NETCoreApp,Version=v7.0). Package Avalonia.iOS 11.0.0-preview5 supports: net6.0-ios16.1

Reply:
@maxkatz6

Reply:
Fixed